src/functions/module_logical.ts

```diff
@@ -70,7 +70,7 @@ export const IF = {
       return { value: "" };
     }
     if (result.value === null) {
-      result.value = "";
```

Comment:
@laa-odoo 
I don't know why it was not simply `return { value: "" }`, included in the if above.
Is it important to preserve the format in this case?

Reply:
the format of the arguments is passed (in GSheet but not in Excel Desktop) while going through IF. 
I think it make sense to display the same representation that you point towards to